mlflow/utils/requirements_utils.py

```diff
@@ -409,3 +417,38 @@ def _get_pinned_requirement(package, version=None, module=None):
             version = version_raw
 
     return f"{package}=={version}"
+
+
+_MismatchPackageInfo = namedtuple(
+    "_MismatchPackageInfo", ["package_name", "installed_version", "requirement"]
+)
+
+
+def _check_pkg_installed_version_satisfy_requirements(req_line):
```

Comment:
```suggestion
def _check_pkg_installed_version_satisfy_requirements(requirement_str):
```

Reply:
```suggestion
def _check_requirement_satisfied(req_line):
```

Can we simply call this function `_check_requirement_satisfied`?